plugins/module_utils/job.py

```diff
@@ -393,3 +393,109 @@ def _ddname_pattern(contents, resolve_dependencies):
             )
         )
     return str(contents)
+
+
+def FixDSName( name_in ):
```

Comment:
I see the C influence in this fragment of code, just change the function name to adhere to this PEP "Function names should be lowercase, with words separated by underscores as necessary to improve readability." https://peps.python.org/pep-0008/#function-and-variable-names

Reply:
just made that change